hackjohn.py

```diff
@@ -140,3 +140,27 @@ def get_trailhead_df():
     mmb_response = requests.post(mmb_url, json=payload)
     print('middleman status code', mmb_response.status_code)
     print(mmb_response.text)
+
+## Notifications using IFTTT
+
+# Set enable_ifttt to True to receive IFTTT notification
+enable_ifttt = True
+event_name = 'hackjohn'
+ifttt_key = 'cJLFuy0CHPBwK0wqndI2g9'
+
+# Create you own IFTTT (https://ifttt.com/create)
+# Select webhooks for this and select receive a web trigger
+# Set event name to match event_name variable above
+# Create a that to respond to the event (IFTTT rich notifications allow click to call options to the reservation line.) You can add 3 values (specified below).
```

Comment:
Does the user have to update the message template to include the values, like:

```
The event named "{{EventName}}" occurred on the Maker Webhooks service

{{Value1}}
{{Value2}}
```

Reply:
They can set it up however they'd like, I set mine to have a rich text notification with {{Value1}} as the message and then tel: {{Value2}} as link action.